### PR TITLE
tidy up

### LINKED ISSUE #8: The /v1 folder has no index.html
This: http://data.sunpy.org/sunpy/v1

url should give a index page so people can see what data is there. It needs to have one generated.

### PR DESCRIPTION
**Pull request by nabobalis**
Removed the 0.8 sample data folder.
Removed the sunkit-sst since it doesn't exist.
Renamed irispy to sunraster
Added bash scripts to create index pages and added that to the travis CI script. Fixes #8
Removed the git lfs setup since the docs say its supported now.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,16 @@
 language: python
 
-before_install:
```
Comment (nabobalis) [inline .travis.yml:3]: Travis says its installed already.

**Review by dstansby — APPROVED** [review]

**Comment by Cadair** [comment]
I would rather not remove the sst data, can we just rename the folder to ndcube or something?

**Comment by nabobalis** [comment]
> I would rather not remove the sst data, can we just rename the folder to ndcube or something?

Those SST files aren't really valid SST files. It is data from the SST but it can't be used to validate anything.

**Review by Cadair — APPROVED** [review]

**Comment by nabobalis** [comment]
Do not merge this.

**Comment by nabobalis** [comment]
Fixed my bad LFS use.

**Pull request merged**